File: JackpotProj_VB/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Timer1.Start()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Timer1.Stop()
    End Sub

    Private Sub Timer1_Tick(sender As Object, e As EventArgs) Handles Timer1.Tick
        SendKeys.Send("1234")
        SendKeys.Send("{ENTER}")
    End Sub
End Class

File: 1.2_JackpotProj_WindowsFormsApp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Threading;
using System.Windows.Forms;

namespace JackpotProj_WindowsFormsApp
{
    public partial class Form1 : Form
    {
        System.Diagnostics.Stopwatch stw = new System.Diagnostics.Stopwatch();
        public Form1()
        {
            InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            timer1.Start();
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            timer1.Stop();
        }

        int input_data = 0;
        private void Timer1_Tick(object sender, EventArgs e)
        {
            Mutex m_mutex = new Mutex();

            if (input_data % 100 == 0)
            {
                textBox1.Text += input_data.ToString();
                textBox1.Text += ' ';
            }
            SendKeys.Send(input_data.ToString());

            SendKeys.Send("{ENTER}");


            label1.Text = input_data.ToString();
            input_data += 1;
        }

    }
}
